content/en/blog/2019/introducing-istioctl-analyze/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Introducing istioctl analyze
+description: Analyze your Istio configuration to detect potential issues and get general insights.
+publishdate: 2019-11-14
+subtitle:
+attribution: David Ebbo (Google)
+keywords: [debugging,istioctl,configuration]
+target_release: 1.4
+---
+
+Istio 1.4 introduces an experimental new tool to help you analyze and debug your clusters running Istio.
+
+[`istioctl analyze`](/docs/reference/commands/istioctl/#istioctl-experimental-analyze) is a diagnostic tool that detects potential issues with your
+Istio configuration, as well as gives general insights to improve your configuration.
+It can run against a live cluster or a set of local configuration files.
+It can also run against a combination of the two, allowing you to catch problems before you
+apply changes to a cluster.
+
+To get started with it in just minutes, head over to the [documentation](/docs/ops/diagnostic-tools/istioctl-analyze/).
+
+## Designed to be approachable for novice users
+
+One of the key design goals that we followed for this feature is to make it extremely approachable.
+This is achieved by making the command useful without having to pass any required complex parameters.
+
+In practice, here are some of the scenarios that it goes after:
+
+- *"There is some problem with my cluster, but I have no idea where to start"*
+- *"Things are generally working, but I'm wondering if there is anything I could improve"*
+
+In that sense, it is very different from some of the more advanced diagnostic tools, which go
+after scenarios along the lines of (taking `istioctl proxy-config` as an example):
+
+- *"Show me the Envoy configuration for this specific pod so I can see if anything looks wrong"*
+
+This can be very useful for advanced debugging, but it requires a lot of expertize before you
```

Comment:
s/expertize/expertise/

Reply:
I think US spelling is with a z, no? :)

Reply:
US does mess up z/c/s a lot, but not in this case :) "expertise" is definitely what you want. "expertize" does exist, but I had never heard it before I saw it here.

Reply:
That's why I despize English ;)